python-sdk/nuimages/utils/utils.py

```diff
@@ -39,3 +42,23 @@ def mask_decode(mask: dict) -> np.ndarray:
     new_mask = mask.copy()
     new_mask['counts'] = base64.b64decode(mask['counts'])
     return cocomask.decode(new_mask)
+
+
+def get_font(fonts_valid: List[str], font_size: int = 15) -> ImageFont:
+    """
+    Check if there is a desired font present in the user's system. If there is, use that font; otherwise, use a default
+    font.
+    :param fonts_valid: A list of fonts which are desirable.
+    :param font_size: The size of the font to set. Note that if the default font is used, then the font size
+        cannot be set.
+    :return: An ImageFont object to use as the font in a PIL image.
+    """
+    # Find a list of fonts within the user's system.
+    fonts_in_sys = matplotlib.font_manager.findSystemFonts(fontpaths=None, fontext='ttf')
+    # Of all the fonts found in the user's system, check if any of them are desired.
+    for font_in_sys in fonts_in_sys:
+        if any(os.path.basename(font_in_sys) in s for s in fonts_valid):
+            return ImageFont.truetype(font_in_sys, font_size)
+
+    # If none of the fonts in the user's system are desirable, then use the default font.
+    return ImageFont.load_default()
```

Comment:
Add a warning using `warnings.warn()` and check that it is only printed at most once.